### PR TITLE
Frontend Spot check remediations

### PR DESCRIPTION
**Pull request by hilvitzs**
### This pull request changes...

- Make better use of utility classes and uswds components
- Make "Skip to Main Content" first selectable element with keyboard tab
- Update secondary nav links to links instead of buttons
- revised cypress tests for landing page

### [AC] This pull request is ready to merge when...

- [ ] Tests have been updated (and all tests are passing)
- [ ] Vulnerability Scan Test (Zap) has no high/critcial CVEs
- [ ] Linting Tests (ESLint for UI and Flake8 for Python)
- [ ] Unit Tests ( including a code coverage test report)
- [ ] Accessibility Tests (Pa11y UI Only)
- [ ] This code has been reviewed by someone other than the original author
- [ ] The experience passes a basic manual accessibility audit (keyboard nav, screenreader, text scaling) OR an exemption is documented
- [ ] The change has been documented
- [ ] Associated OpenAPI documentation has been updated
- [ ] Changelog is updated as appropriate

### [DoD] This feature is done when...

- [ ] Design has approved the experience
- [ ] Product has approved the experience

### CODE REVIEW

**Comment by hilvitzs** [comment]
After discussions with @carltonsmith, @shubhi-raft, and @lauraGgit we are going to remove trussworks from our project. It is just not quite mature enough to be used currently. Here is the issue: https://github.com/raft-tech/TANF-app/issues/274.

File: tdrs-frontend/src/theme/_global.scss [hunk 1]
```diff
@@ -1,3 +1,20 @@
 @import 'include-media/dist/include-media';
 
+// $theme-image-path: '~uswds/src/img';
+// $theme-font-path: '~uswds/src/fonts';
```
Comment (RafterGit) [inline tdrs-frontend/src/theme/_global.scss:4]: @hilvitzs  Does this commented out code serve a purpose or can it be deleted?

**Review by RafterGit — CHANGES_REQUESTED** [review]

**Review by RafterGit — APPROVED** [review]

**Review by lauraGgit — COMMENTED** [review]
The deployed version currently has the logo in the footer, but the local version from this branch does not. Can you add in that code here and it will be good to go?

**Comment by hilvitzs** [comment]
I'm not sure how the deployed was different than the branch but I just pushed with the ACF logo replacing the placeholder.

**Review by lauraGgit — APPROVED** [review]

**Pull request merged**